b2sdk/file_version.py

```diff
@@ -197,6 +197,19 @@ def _type(self):
         assert m, self.id_
         return self._FILE_TYPE[int(m.group(1))]
 
+    def get_content_sha1(self) -> Optional[Sha1HexDigest]:
+        """
+        Get the file's content SHA1 hex digest from the header or, if its absent,
+        from the file info.  If both are missing, return None.
+        """
+        if self.content_sha1 and self.content_sha1 != "none":
+            return self.content_sha1
+        elif LARGE_FILE_SHA1 in self.file_info:
+            return self.file_info[LARGE_FILE_SHA1]
```

Comment:
Second condition can be replaced with `self.file_info.get(LARGE_FILE_SHA1)`.

Reply:
Actually, dict contains `str`, result should be `Sha1HexDigest`, so it's casted here and returned.